package-drone/src/main/resources/de/dentrassi/pm/jenkins/Messages.properties

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2015 IBH SYSTEMS GmbH.
```

Comment:
This file seems all new. So the first line would be: `Copyright (c) 2017 Nikolas Falco.`